### PR TITLE
Update ACA-Py dependencies

### PR DESCRIPTION
**Pull request by WadeBarnes**
Resolves: #28

### CODE REVIEW

File: redis_events/pyproject.toml
Comment (jamshale) [inline redis_events/pyproject.toml:18]: My repo-manager script, which I think will become more important as this repo grows, works off of empty lines between sections. I don't think it would handle the comments very well either because it sorts.

But I can change the logic a bit to handle things like this better. I think this approach makes sense. We'll have to do this for the other plugins as well but can merge this for now and then do the rest if this is successful.

**Comment by jamshale** [comment]
I think this will work good. I'll just add a couple reviewers to get some other eyes on it.

**Review by dbluhm — APPROVED** [review]

**Comment by WadeBarnes** [comment]
The tests on the `connection_update` plugin are failing due to an expired certificate on https://indy.igrant.io/

**Comment by WadeBarnes** [comment]
The `basicmessage_storage` plugin is affected by the same issue.

**Comment by WadeBarnes** [comment]
Ready to go.  Though we need to figure out what to do with the failing tests.  We either need to get `igrant.io` to update the certificate or switch to a different ledger like `http://test.bcovrin.vonx.io/`.  Open to suggestions.

**Comment by swcurran** [comment]
Agree that using BCovrin Test is the right thing to do for this.

**Comment by WadeBarnes** [comment]
Switching to `http://test.bcovrin.vonx.io/` fixed the issues with the failing integration tests.

File: multitenant_provider/integration/pyproject.toml
Comment (jamshale) [inline multitenant_provider/integration/pyproject.toml:8]: Think this was missed

File: multitenant_provider/pyproject.toml
Comment (jamshale) [inline multitenant_provider/pyproject.toml:16]: here too

File: multitenant_provider/integration/pyproject.toml
Comment (WadeBarnes) [reply multitenant_provider/integration/pyproject.toml:8]: The `multitenant_provider` didn't like it when it was updated to a newer version of ACA-Py, the tests failed, so I reverted it back to the version that worked.
See:
- https://github.com/hyperledger/aries-acapy-plugins/actions/runs/7090862684/job/19298755060
- https://github.com/hyperledger/aries-acapy-plugins/actions/runs/7090862685/job/19298755070

File: multitenant_provider/pyproject.toml
Comment (WadeBarnes) [reply multitenant_provider/pyproject.toml:16]: The `multitenant_provider` didn't like it when it was updated to a newer version of ACA-Py, the tests failed, so I reverted it back to the version that worked.
See:
- https://github.com/hyperledger/aries-acapy-plugins/actions/runs/7090862684/job/19298755060
- https://github.com/hyperledger/aries-acapy-plugins/actions/runs/7090862685/job/19298755070

File: multitenant_provider/integration/pyproject.toml
Comment (jamshale) [reply multitenant_provider/integration/pyproject.toml:8]: OK. I can have a look after this is merged.

**Pull request merged**
